Taskfile.yml

```
@@ -37,7 +37,7 @@ tasks:
   setup-win:
     platforms: [ windows ]
     cmds:
-      - cmd: powershell rm -r {{.HG_REPO_DIR}}/sena-3
+      - cmd: powershell if (test-path {{.HG_REPO_DIR}}/sena-3) { rm -r {{.HG_REPO_DIR}}/sena-3 }
```

```suggestion
      - cmd: powershell "if (test-path {{.HG_REPO_DIR}}/sena-3) { rm -r {{.HG_REPO_DIR}}/sena-3 }"
```
need to wrap that part in quotes otherwise task blows up trying to parse it for some reason.

---

And that's why I needed someone with a Windows machine to test it. Thanks.